libcxx/test/std/experimental/simd/simd.reference/reference_assignment.pass.cpp

```diff
@@ -29,11 +29,9 @@ struct CheckSimdReferenceAssignmentHelper {
   void operator()() const {
     if constexpr (std::is_assignable_v<T&, U&&>) {
       ex::simd<T, SimdAbi> origin_simd([](T i) { return i; });
-      for (size_t i = 0; i < origin_simd.size(); ++i) {
-        static_assert(noexcept(origin_simd[i] = static_cast<U>(i + 1)));
-        origin_simd[i] = static_cast<U>(i + 1);
-        assert(origin_simd[i] == static_cast<T>(std::forward<U>(i + 1)));
-      }
+      static_assert(noexcept(origin_simd[0] = static_cast<U>(5)));
```

Comment:
I don't think this actually reduces the compilation time since there's no additional instantiations being performed here. This is just code running at runtime, and the runtime of the test is not the issue. Did I miss something?